dask/multiprocessing.py

```diff
@@ -143,14 +143,8 @@ def get_context():
         if config.get("multiprocessing.context", None) is not None:
             warn(_CONTEXT_UNSUPPORTED, UserWarning)
         return multiprocessing
```

Comment:
On Windows the default (and only supported) context is `"spawn"`. Given this, can we drop this block?